src/ast/nodes/CallExpression.ts

```diff
@@ -135,6 +135,8 @@ export default class CallExpression extends NodeBase implements DeoptimizableEnt
 		for (const argument of this.arguments) {
 			if (argument.hasEffects(options)) return true;
 		}
+		debugger;
```

Comment:
Debugger statement here

Reply:
I would say this could go either into the 0.66 or the 1.0. But give us a little time, got a lot of stuff around my head at the moment and I want to do a thorough review of this and also do some performance tests on large code-bases.

Reply:
Sure. I'm optimistic about the perf aspect, but I also want to see what happens when it's used on a large codebase.